proto/vendorinfo/query.proto

```diff
@@ -3,13 +3,43 @@ package zigbeealliance.distributedcomplianceledger.vendorinfo;
 
 import "google/api/annotations.proto";
 import "cosmos/base/query/v1beta1/pagination.proto";
+import "vendorinfo/vendor_info.proto";
 // this line is used by starport scaffolding # 1
+import "gogoproto/gogo.proto";
 
 option go_package = "github.com/zigbee-alliance/distributed-compliance-ledger/x/vendorinfo/types";
 
 // Query defines the gRPC querier service.
 service Query {
-    // this line is used by starport scaffolding # 2
+    // Queries a vendorInfo by index.
+	rpc VendorInfo(QueryGetVendorInfoRequest) returns (QueryGetVendorInfoResponse) {
+		option (google.api.http).get = "/dcl/vendorinfo/vendor/{vendorID}";
+	}
+
+	// Queries a list of vendorInfo items.
+	rpc VendorInfoAll(QueryAllVendorInfoRequest) returns (QueryAllVendorInfoResponse) {
+		option (google.api.http).get = "/dcl/vendorinfo/vendors";
+	}
+
+// this line is used by starport scaffolding # 2
+}
+
+message QueryGetVendorInfoRequest {
+	  uint64 vendorID = 1;
+
+}
+
+message QueryGetVendorInfoResponse {
+	VendorInfo vendorInfo = 1 [(gogoproto.nullable) = false];
```

Comment:
(possibly for the next PR) please don't forget to follow the approach for NotFound entries as it has been discussed
#138 makes sense to use as an example.